ros_gz/package.xml

```diff
@@ -4,7 +4,7 @@
   <!-- TODO: Make this a metapackage, see
        https://github.com/ros2/ros2/issues/408 -->
   <name>ros_gz</name>
-  <version>0.247.0</version>
+  <version>0.245.0</version>
```

Comment:
I'm not actually sure what we want to do here.  `245` is the latest release in iron and `247` is the latest in humble.

I think we want to keep the newer version and release it.  @ahcorde what do you think

Reply:
I think we should modify the version to `1.X`, `2.X`, ... sometimes is hard to find a version which is not already released

Reply:
According to https://github.com/gazebosim/ros_gz/blob/ros2/RELEASING.md#versioning, the versions for Iron should start with `0.254`. Do we want to keep that convension?

Reply:
I've reverted the change.